packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -115,7 +115,7 @@ void main() {
       ),
     );
 
-    await tester.fling(find.text('X'), const Offset(0.0, -300.0), 1000.0);
+    await tester.fling(find.text('X'), const Offset(0.0, 600.0), 1000.0);
```

Comment:
I'm running this PR through internal testing now, I am concerned it may be breaking, but hopeful it won't be.

Reply:
@darrenaustin  actually approved this (so I won't make changes to turn it into parameter based direction) 
Can you please take a look if this breaks any internal tests?

Reply:
Out internal presubmit tester is having some issues, I think this is ok to land though.